config/deploy/staging.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# server-based syntax
+# ======================
+
+# Grab ruby version from project home directory. It is needed for deployment.
+set :rvm_ruby_version, File.read(File.expand_path('./../../.ruby-version', __dir__)).strip
+
+set :server_ip, Rails.application.credentials.dig(:deploy, :server)
+set :user, Rails.application.credentials.dig(:deploy, :user)
```

Comment:
Does `config/deploy/staging.rb` "inherit" variables from `config/deploy.rb`? If so, no need to duplicate the `set :user` call.

Reply:
Yes, it can be inherited.   Update the code